### PR TITLE
gh-285 Fix Production bundle error

### PR DESCRIPTION
**Pull request by ghillert**
- Temporarily upgrade to `tixif-ngx-busy`
- Upgrade to Angular Cli `1.3.0 RC3`

Resolves https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/311

### CODE REVIEW

**Comment by cppwfs** [comment]
Once this PR is merged.  We will need to put a note in the slack room telling folks to 
1) delete node_modules 
2) npm install
3) ng build --prod

File: ui/package.json [hunk 1]
```diff
@@ -11,39 +11,42 @@
...
+    "@angular/forms": "^4.2.4",
+    "@angular/http": "^4.2.4",
+    "@angular/platform-browser": "^4.2.4",
+    "@angular/platform-browser-dynamic": "^4.2.4",
+    "@angular/router": "^4.2.4",
     "core-js": "^2.4.1",
+    "rxjs": "^5.4.2",
+    "zone.js": "^0.8.14",
+
+    "tixif-ngx-busy": "0.0.5",
```
Comment (cppwfs) [inline ui/package.json]: Why did we switch from angular2-busy which is at 2.0.4 to tixif-ngx-busy with is at 0.0.5 ?

**Comment by cppwfs** [comment]
As a separate issue we probably want to update our build instructions on the readme.  it says to `mvn clean package install`  Which causes 2 compodoc builds. probably want switch to `npm clean install`

File: ui/src/polyfills.ts [hunk 2]
```diff
@@ -32,21 +31,21 @@ import 'core-js/es6/string';
 import 'core-js/es6/array';
 // import 'core-js/es6/regexp';
 // import 'core-js/es6/map';
+// import 'core-js/es6/weak-map';
```
Comment (ghillert) [inline ui/src/polyfills.ts]: They are part of the Angular CLI files. Thus I would rather leave the Angular provided files as unmodified as possible.  That way we know also all the relevant polyfills and can enabled them as needed we start testing across more browsers.

File: ui/tslint.json [hunk 3]
```diff
@@ -80,6 +86,7 @@
     ],
     "radix": true,
     "semicolon": [
+      true,
```
Comment (ghillert) [inline ui/tslint.json]: Sounds good - please create a follow up issue as it would affect a large number of files.

File: ui/package.json (hunk 1 above)
Comment (ghillert) [inline ui/package.json]: Issued is related to https://github.com/devyumao/angular2-busy/issues/55

The temporary fix is to use https://www.npmjs.com/package/tixif-ngx-busy for the time being.

**Comment by cppwfs** [comment]
LGTM.  Rebased Merged.

**Pull request closed without merging**